### ISSUE TITLE
upcoming merge deprecations

### ISSUE THREAD

**Issue by bgentry** [bgentry/ember-apollo-client#172]
noticed these in the test logs on canary:

```
DEPRECATION: Use of `merge` has been deprecated. Please use `assign` instead
```

This should be a pretty straightforward change, would love if anybody wants to help with it.

**Comment by christophermlne** [comment]
@bgentry i've been thinking about how i could give back to this project. this seems like a good place to start. 

i can take this one

**Comment by bgentry** [comment]
Resolved by #174.
